packages/application-generic/src/usecases/digest-filter-steps/digest-filter-steps.usecase.ts

```diff
@@ -1,33 +1,18 @@
 import { Injectable, Logger } from '@nestjs/common';
 import { NotificationStepEntity } from '@novu/dal';
-import { DigestTypeEnum, StepTypeEnum } from '@novu/shared';
+import { StepTypeEnum } from '@novu/shared';
 
 import { DigestFilterStepsCommand } from './digest-filter-steps.command';
-import { DigestFilterStepsRegular } from './digest-filter-steps-regular.usecase';
-import { DigestFilterStepsTimed } from './digest-filter-steps-timed.usecase';
 
 const LOG_CONTEXT = 'DigestFilterSteps';
 
 // TODO; Potentially rename this use case
 @Injectable()
 export class DigestFilterSteps {
-  constructor(
-    private filterStepsRegular: DigestFilterStepsRegular,
-    private filterStepsTimed: DigestFilterStepsTimed
-  ) {}
-
   public async execute(
     command: DigestFilterStepsCommand
   ): Promise<NotificationStepEntity[]> {
-    const action: DigestFilterStepsRegular | DigestFilterStepsTimed =
```

Comment:
This code is not needed anymore, since we want to only make a single decision on digest merged or not in the digest add job phase. Instead of doing it multiple times.